RecoBTag/TensorFlow/python/pfDeepDoubleCvBJetTags_cfi.py

```diff
@@ -0,0 +1,21 @@
+import FWCore.ParameterSet.Config as cms
+
+pfDeepDoubleCvBJetTags = cms.EDProducer('DeepDoubleXTFJetTagsProducer',
+  src = cms.InputTag('pfDeepDoubleXTagInfos'),
```

Comment:
shouldn't this be just a clone of existing config ```pfDeepDoubleCvBJetTags = pfDeepDoubleBvLJetTags.clone(flavor = 'CvB')```
?
This will avoid replication of what is already encoded in the fillDescriptions.